### PR TITLE
More conditional and loop snippets

### PR DESCRIPTION
**Pull request by EmilyGraceSeville7cf**
- snippets for comparisons in `if` and `while`
- snippets for iterating over ranges and files for `for`
- simplify snippet names for builtins
- remove snippets for list, table and range creations as they are simple constructs to type
- use `${<number>:command ...}` instead of `${<number>:block}` and `${<number>:closure}`

### CODE REVIEW

File: snippets/nushell.json [hunk 1]
```diff
@@ -317,7 +289,7 @@
             "}"
         ]
     },
-    "def-env builtin": {
+    "def-env": {
```
Comment (fdncred) [inline snippets/nushell.json:292]: `def-env` is deprecated in favor of `def --env` now

**Comment by fdncred** [comment]
Good to see you back. Thanks!

**Comment by fdncred** [comment]
Can you add a `def --wrapped` in the same manner that you just changed `def --env`? If we can get that in, I'll land this.

**Comment by EmilyGraceSeville7cf** [comment]
Done. :)

**Review by fdncred — APPROVED** [review]
Thanks

**Pull request merged**